projects/igniteui-angular/src/lib/combo/combo.component.ts

```diff
@@ -1324,11 +1324,13 @@ export class IgxComboComponent extends DisplayDensityBase implements IgxComboBas
      * @hidden @internal
      */
     public handleClearItems(event: Event): void {
-        this.deselectAllItems(true, event);
-        if (this.collapsed) {
-            this.getEditElement().focus();
-        } else {
-            this.focusSearchInput(true);
+        if (!this.disabled) {
```

Comment:
Could this be moved above the other logic w/ a `return` statement?:
```
if (this.disable) {
    return;
}
...
```

I know it would change the logic and the event will not be stopped from propagating, but the click handler for opening should still have a `disabled` check, so we should be fine

Reply:
You are right that onInputClick is called, where we already have event.stopPropogation().